package.json

```diff
@@ -42,7 +44,7 @@
   },
```

Comment:
As stated in https://babeljs.io/docs/usage/babelrc/, instead of a `.babelrc` file, you can put your settings in _package.json_. Insert this block above `"scripts"`:
```json
  "babel": {
    "presets": ["es2015"]
  },
```
I think I prefer this approach because it is more visible and keeps things in one place.